src/main/java/com/commercetools/sync/services/TypeService.java

```diff
@@ -26,4 +32,55 @@
      */
     @Nonnull
     CompletionStage<Optional<String>> fetchCachedTypeId(@Nonnull final String key);
+
```

Comment:
please remove all the `potentially` from the Javadocs as per @lojzatran's recommendation.

`defined in a potentially injected {@link SphereClient}` -> `defined in an injected {@link SphereClient}`